packages/oas-form/src/utils.js

```diff
@@ -246,9 +242,6 @@ function computeDefaults(_schema, parentDefaults, rootSchema, rawFormData = {},
     }
 
     return computeDefaults(refSchema, defaults, rootSchema, formData, includeUndefinedValues);
```

Comment:
Discriminators uses this, but I never really bothered to check what it was for? If the discriminator tests run I probably feel fine with this change.

Reply:
This is what compiles the original default `formData` on load.